apps/web/src/pages/Skills/SkillPortfolioPage.tsx

```diff
@@ -71,13 +71,13 @@ const SkillLibrary = ({ userSkills, skills }: SkillLibraryProps) => {
         url: paths.profileAndApplications(),
       },
       {
-        label: intl.formatMessage(navigationMessages.skillLibrary),
+        label: intl.formatMessage(navigationMessages.skillPortfolio),
         url: paths.skillLibrary(),
```

Comment:
I think it might be best to update the `skillLibrary()` path as well to `skillPortfolio()` to keep things straight?

Reply:
Great suggestion, renamed in [rename skill library path](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11616/commits/5ab2d1ede011beff263df9c95bb0e2d604dd7e0c)